### PR TITLE
Fix flake8 warnings, release 1.2

### PR DESCRIPTION
**Pull request by alexeyklyukin**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -24,6 +24,7 @@ def read_version(package):
         exec(fd.read(), data)
     return data['__version__']
 
+
```
Comment (avaczi) [inline setup.py]: a substantial change ;)

**Comment by avaczi** [comment]
:+1:

**Comment by alexeyklyukin** [comment]
👍

File: setup.py (hunk 1 above)
Comment (alexeyklyukin) [inline setup.py]: flake8 was not happy about the lack of the extra line after the function definition

**Pull request merged**